libqtile/widget/__init__.py

```diff
@@ -27,6 +27,7 @@
     "AGroupBox": "groupbox",
     "Backlight": "backlight",
     "Battery": "battery",
+    "AccessDots": "accessDots",
```

Comment:
The first string is the name of the widget class so needs to match what you call with the widget.